.env.example

```
@@ -0,0 +1,7 @@
+# Server Configuration
+HOST=0.0.0.0
+PORT=3000
+
+# CORS Configuration
+CORS_ORIGIN=*
+CORS_CREDENTIALS=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)